packages/bpk-docs/src/layouts/links.js

```diff
@@ -608,4 +608,15 @@ export default [
       },
     ],
   },
+  {
+    id: 'STYLE_GUIDE',
+    category: 'Style guide',
+    sort: true,
+    hero: {
+      url: `/${DesignTokensImage}`, // TODO
```

Comment:
Will be fixed in the subsequent card to flesh out the page and hero image component.